### PR TITLE
Cleaning up ServiceModel string resources.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* Removed resource strings related to configuration files.
* Removed resource strings related to reliable messaging and reliable sessions.

### CODE REVIEW

**Comment by iamjasonp** [comment]
Good first pass - how'd you go through these to determine what was used/not? Did you use some kind of tool, or manually? 

It seems like there are a number of strings (e.g. SecurityBindingElementCannotBeExpressedInConfig, DeliveryAssuranceRequired) that weren't taken out. 

No objections to merging what you have now and then we can continue to prune the list as time goes on.

File: src/System.Private.ServiceModel/src/Resources/Strings.resx [hunk 1]
```diff
@@ -3108,51 +2964,24 @@
   <data name="UnknownSequenceFaultReason" xml:space="preserve">
     <value>The value of wsrm:Identifier is not a known Sequence identifier.</value>
   </data>
```
Comment (roncain) [inline src/System.Private.ServiceModel/src/Resources/Strings.resx]: The "Sequence" and "RM" messages above can probably also be removed at some point

**Comment by roncain** [comment]
LGTM

**Comment by StephenBonikowsky** [comment]
I had to touch this file to add a resource for my previous PR and that's when I saw a bunch of the resources for config, so I thought any cleanup done to this file would be positive, @mconnew mentioned that we don't want to remove any resource strings that we might eventually pull in so then I hunted for Reliable Messaging and Reliable Sessions strings which I don't think we will ever pull in. This isn't a complete cleanup by any means, just some of the more obvious ones.

**Pull request merged**